### PR TITLE
Update wcag-ruleset-review-process.md

### PR DESCRIPTION
**Pull request by WilcoFiers**
## How this improves the process

The proposed process is different from the current ACT rule review process in the following ways:

1. The ACT Rules community group website is dropped. Rules are published directly to the WAI website, but with an indication that the rule is "proposed". Implementation tracking too is done on the WAI website. Any guidance on rule writing, process descriptions, and the definition of done is used directly in the github repository.

2. ACT TF members will have an active role in updating rules. In their review, TF members propose changes. Instead of handing feedback to the rule author, the liaison is responsible for drafting any necessary changes themselves.

3. The validation process now involves both members from the TF and the CG, instead of being exclusive to the community group.

4. Instead of having two separate but connected processes, one managed by the CG and the other by the TF, a single process exists that is maintained by the TF.

5. There is no longer a separate CFC step in the Task Force. The survey is essentially the CFC; and not answering the survey is akin to not responding on a CFC.

6. The "final call" step in the CG is renamed to "Call for review", as this step is no longer the "final" step in the process.

### CODE REVIEW

File: wcag-ruleset-review-process.md
Comment (tbostic32) [inline wcag-ruleset-review-process.md:23]:
```suggestion
The pull request description must include information on the type of change the pull request makes and what happens in the "call for review" phase. There are three options:
```
I don't think this is a correct use of a comma (I do this a lot myself and am trying to get out of the habit.). I think the same thing applies to the list below.

File: wcag-ruleset-review-process.md
Comment (tbostic32) [inline wcag-ruleset-review-process.md:25]: These might actually need to be:
`New rules, and significant changes that impact multiple rules, should have a **two week** call for review.`

Not the best at grammar so ignore if your confident.

File: wcag-ruleset-review-process.md
Comment (john-urbilog) [inline wcag-ruleset-review-process.md:56]: This survey will be open for at least 5 **working** days

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:80]:
```suggestion
For the ACT Task Force agenda, public feedback will be prioritised over review of proposed rules. This ensures responsiveness to the wider community. The ACT Task Force should respond to questions within 5 work days. This may be a reply which indicates when an answer is expected.
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:7]:
```suggestion
Rule writing, reviewing, approving, and maintaining is a joint work between the ACT TF and the ACT Rules Community Group through regular joint meetings. Any other groups interested to contribute rules to the WCAG ruleset should contact the co-facilitator(s) of the ACT Task Force. The ACT publication process is designed to publish high-quality ACT rules to the W3C website. 
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:11]:
```suggestion
When creating a new rule, or updating a rule, there are five steps to go through. This process involves the combined effort of the ACT Task Force and the ACT Rules Community Group. 
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:17]:
```suggestion
5. Publication approval
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:62]:
```suggestion
### 5. Publication Approval
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [reply wcag-ruleset-review-process.md:25]: Or I think they could be:

```suggestion
New rules and significant changes that impact multiple rules should have a **two week** call for 
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:34]: Shall we make explicit how these approvals to comments should be recorded?

Reviewing and approving a pull request may be pretty straight forward if you are familiar with the process. Approving a comment can be done in multiple ways: replying to the comment in GitHub, sending an email, in a follow-up chat...

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:70]:
```suggestion
2. If AG requests editorial changes, a [draft proposal](#1-draft-proposal) is created. Once merged, the updated rule is republished on the WAI website with the "proposal" indicator removed. This skips the implementations or validation steps.
```

File: wcag-ruleset-review-process.md
Comment (daniel-montalvo) [inline wcag-ruleset-review-process.md:72]:
```suggestion
3. If AG requests non-editorial, the liaison is tasked to [draft proposal](#1-draft-proposal).
```

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:117]:
```suggestion
2. All input rules of a composite rule must also have at least one implementation for which the previous requirement is true.

## Editorial Changes

Parts of the process can be bypassed when changes are editorial. If the author believes a change is editorial, this must be indicated in the PR description. If there is disagreement with the reviewers about this, a change should not be considered editorial. Use the following indicators to decide if a change is editorial:

- An editorial change does not add new content. New expectations, test cases, assumptions, accessibility support cases, background info, etc. are not editorial.
- Editorial changes do not change the meaning of the rule, except if for issues where the intended meaning is obvious, such as typos in role names
- Test cases are not changed, except for trivial fixes that are unlikely to impact an implementation.
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:40]:
```suggestion
### 3. Call for implementation

When a pull request is merged, the rule is published on the WAI website with a clear indicator that the rule, or the update to the rule is a "proposal". Additionally, metadata for the rule is published, so that implementors of ACT rules can create an implementation report for the proposed rule.
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:80]: Emma suggested:

```suggestion
For the ACT Task Force agenda, public feedback will be prioritised over review of proposed rules. This ensures responsiveness to the wider community. The ACT Task Force should respond to questions within 5 work days. This may be a brief reply that indicates when a full response is expected.
```

File: wcag-ruleset-review-process.md
Comment (Jym77) [inline wcag-ruleset-review-process.md:42]: Back in https://github.com/act-rules/act-rules.github.io/issues/577 and https://www.w3.org/2019/06/27-act-r-minutes.html#item03 the CG decided that 3 implementations were needed. I couldn't find when we decided to go down to only 1 implementation.
Apparently, we also decided for a CG CFC before sending to the TF, which never happened… So I guess this was maybe changed at a later point, but when?

File: wcag-ruleset-review-process.md
Comment (Jym77) [inline wcag-ruleset-review-process.md:76]: For the WCAG ruleset, I guess we'll also be getting feedback sent to the AG or the CG, rather than the TF (typically, as issues on their respective Github repos). Do we need to duplicate these issues on the TF repo?
I'm notably concerned with the CG repo where there is next to no triaging happening on the issues, and where we might end up just fixing the issues rather than moving them to the TF repo (which requires digging out to remember if a rule is a WCAG published one or not).

File: wcag-ruleset-review-process.md
Comment (annethyme) [inline wcag-ruleset-review-process.md:5]: Much of the tasks in this process are delegated to ACT Task Force, which is a sub-group of AGWG. 
--> 
Much of the tasks in this process are delegated to ACT Task Force (ACT TF), which is a sub-group of AGWG.

File: wcag-ruleset-review-process.md
Comment (Jym77) [inline wcag-ruleset-review-process.md:84]: I guess some of the oldest published rules are reaching the 1 year mark (https://github.com/act-rules/act-rules.github.io/issues/1120 doesn't track publication date, but is from pre-covid times…)
Is this review process planned already for these rules?

File: wcag-ruleset-review-process.md
Comment (annethyme) [inline wcag-ruleset-review-process.md:31]: Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules community group...
-->
Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules Community Group... (capitalization)

File: wcag-ruleset-review-process.md
Comment (Jym77) [inline wcag-ruleset-review-process.md:86]: I'm not sure who the "rule's liaison" exactly is (since it is different from the ACT facilitator).
It is a bit of a strong requirement to give people a 10 work days deadline.
At https://github.com/w3c/wcag-act/pull/515/files#diff-1c6fbe9f1dcc2b7aeae3ce3d63c69e16cb5039c91b5b49a5ca0b8516897de840R80 it is a global deadline for the TF to answer, which makes sense, but here it seems to be an individual deadline. The way this sentence is written, it feels like  "that rule's liaison" is a well defined pre-existing person (the CG member who sent the rule to the TF, maybe?) If so, it is a strong requirement to ask them to be able to act within 10 work days at some unspecified point in the future.

File: wcag-ruleset-review-process.md
Comment (annethyme) [inline wcag-ruleset-review-process.md:7]: Is "interested to contribute" correct, og should it be "interested in contributing"?
I at least found it a bit hard to read. 

Found this on the subject (https://jakubmarian.com/interested-in-doing-vs-interested-to-do-in-english/): 
`“Interested to” can be used only with verbs of perception and “knowing”, such as:see, hear, read, learn, know, find out, …`

**Review by annethyme — COMMENTED** [review]
As an "outsider" to what has been discussed on the process in this round, it looks really good, well-described and with one relatively easy to understand process from draft rule to official publication. Well done! 
The only more general questions I have after intitially reading this is where the boundary between CG and TF is? And these "liaisons" where do they come from? (I unfortunately don't have time to read through another time to see if I could actually find the answers in the text, if reading more carefully).
I have noted down a few editorial comments as well.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:31]: (Just making this a little easier to pull in)

```suggestion
Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules Community Group mailing list with the text "[for review]" in the subject. The e-mail must include a link to the pull request, and an indication of the duration and end date of the call for review. 
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:76]: I don't quite know the practical implications yet. We should avoid duplicate issues, so we're going to need to funnel everything to the same place. We might just end up with a shared repo between TF / CG for rules and definitions.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:84]: Yeah, the TF is already doing this.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:86]: Liaisons can change. If whoever's the current liaison can't do it in 10 days, someone else needs to take over.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:36]:
```suggestion
    1. If the changes are editorial, the pull request can be merged any time after the review period.
    2. If the changes are not editorial, repeat step 1, followed by a **one week** call for review.
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:70]:
```suggestion
2. If AG requests editorial changes, a [drafts proposal](#1-draft-proposal) is created by an ACT facilitator. Once merged, the updated rule is republished on the WAI website with the "proposal" indicator removed. This skips the implementations or validation steps.
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:86]: Should not use "review" maybe "check-in".

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:104]: Heading on what a liaison is.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:100]: ... not complete with cantTell.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:34]: Need to clarify "approval" is on a pull request.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:5]:
```suggestion
Much of the tasks in this process are delegated to ACT Task Force (ACT TF), which is a sub-group of AGWG. ACT TF carries out initial review and prepares proposals for AGWG, which has the final say as the maintainer of WCAG and the authoritative body for interpretation.
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:56]: Working day is more commonly used to refer to the hours of the day in which someone works, whereas work day is the actual day. I think this should be work day. Leaving it as is.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [reply wcag-ruleset-review-process.md:42]: The ACT TF added its own requirements for implementations in August 2020. Here's the PR of it: #472. With this proposal the CG now adopts that requirement.

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:125]:
```suggestion
- Test cases are not changed, except for trivial fixes that are unlikely to impact an implementation.

## Liaisons

A liaison is a member of the ACT TF or CG, appointed to for specific rule maintenance activities. Preferably this is one of the authors of the rule.

```

File: wcag-ruleset-review-process.md
Comment (maryjom) [inline wcag-ruleset-review-process.md:7]: Suggest "...joint work **effort** between..."

Comment (maryjom) [reply wcag-ruleset-review-process.md:7]: Unless a new co-facilitator was found, "co-facilitators" should be  "facilitator."

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:82]:
```suggestion
## Annual Check
```

File: wcag-ruleset-review-process.md
Comment (WilcoFiers) [inline wcag-ruleset-review-process.md:86]:
```suggestion
Every time a rule is [validated](#4-validation), it is checked to make sure it is up to date. This counts as an annual check, so that only rules that have not been surveyed in 12 months need to go through a separate check. If a rule has not been checked for 12 months, an ACT facilitator opens an issue for it, and assigns it to that rule's liaison. The liaison has 10 work days to check if the rule needs to be updated. If the rule needs to be updated, the liaison has another 10 work days to create a [draft proposal](#1-draft-proposal).
```

**Review by daniel-montalvo — APPROVED** [review]
Just minor suggestions.

**Pull request merged**
